packages/components/ng-ovh-payment-method/package.json

```diff
@@ -38,7 +38,9 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/ng-ovh-payment-method' --include-dependencies -- yarn run dev:watch"
   },
   "dependencies": {
-    "@adyen/adyen-web": "3.20.0",
+    "@adyen/adyen-web": "^3.20.0",
+    "@ovh-ux/ovh-payment-method": "^0.0.0",
+    "@ovh-ux/ovh-reket": "^0.0.0",
```

Comment:
I think this packages should also be moved from `packages/manager/modules` to `packages/components` as it has to be public in order to be used outside of the manager monorepo.